server/src/transport/http.rs

```diff
@@ -1,39 +1,117 @@
-use std::convert::Infallible;
+use http::Method;
+use jsonrpsee_core::{http_helpers::read_body, GenericTransportError};
+
+use crate::{
+	middleware::rpc::{RpcService, RpcServiceBuilder, RpcServiceCfg, RpcServiceT, TransportProtocol},
+	server::{handle_rpc_call, Params, Settings},
```

Comment:
Seeing the two together; how do we explain the difference between `Params` and `Settings` (both sound like the same thing); maybe some naming tweak can better differentiate?

Reply:
Settings are "static configuration" of the server.
Params contains "static configuration" + a few connection related things such as connection_id, conn_permit etc.

Reply:
Removed now, I think it's easier to understand now